samples/batch/transcription-enabled-storage/Connector/Serializable/TextAnalytics/TextAnalyticsDocument.cs

```diff
@@ -9,16 +9,16 @@ namespace Connector.Serializable.TextAnalytics
 
     public class TextAnalyticsDocument
     {
-        public TextAnalyticsDocument(string id, IEnumerable<TextAnalyticsSentence> sentences, IEnumerable<TextAnalyticsEntity> entities)
+        public TextAnalyticsDocument(string id, TextAnalyticsConfidenceScores confidenceScores, IEnumerable<TextAnalyticsEntity> entities)
```

Comment:
is it a single score and list of scores?

Reply:
it's not a list, but one score for positive, one for negative, and one for neutral.
That's why I called it confidenceScores, but it's not a list